aries-site/src/pages/components/accordion.mdx

```diff
@@ -0,0 +1,38 @@
+import { Example } from '../../layouts';
+import { AccordionExample } from '../../examples';
+
+<Example
+  docs="https://v2.grommet.io/accordion#props"
+  designer="https://designer.grommet.io/accordion?id=HPE-design-system-hpedesignsystem-hpe-com&mode=edit"
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/accordion/AccordionExample.js"
+  figma="https://www.figma.com/file/UvTBUQdhk07wV79nijsrtr/HPE-Accordion-Component?node-id=151%3A1"
+>
+  <AccordionExample />
+</Example>
+
+## Guidance
+
+When seeking to provide maximum content in limited, vertical space, an accordion is a good choice.
+
+An accordion is a group of vertically stacked collapsible buttons with title headers. The titles give the user a high-level idea of what content will be disclosed when an accordion panel opens.
+
+Accordions help organize information, which helps the user consume the information they need at a faster pace.
+
+### About Accordion
+
+Accordion has two states:
+
+- Collapsed
+- Expanded
+
+The chevron icon is used to identify the expand or collapse action while the entire header can be clicked to expand or collapse content.
```

Comment:
This was previously bolded. If we want to keep the bold (I'm okay with it being removed as well):
```suggestion
**The chevron icon** is used to identify the expand or collapse action while the entire header can be clicked to expand or collapse content.
```